### ISSUE TITLE
ranks issue

### ISSUE THREAD

**Issue by PhanaticD** [Bimmr/McInfected#156]
## What's Currently Happening

players ranks arent showing, errors in logs

----
## Steps to Reproduce (for bugs)

1.
dunno
2.

3.

----
## Error Code
```
<!--- Please put the error in between the ``` tags -->  
[04:54:59] [Server thread/ERROR]: Could not pass event McInfectedPlayerJoinEvent to McInfectedRanks v1.2.3
java.lang.StackOverflowError: null
    at com.mysql.jdbc.ConnectionImpl.getEncodingForIndex(ConnectionImpl.java:2730) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.Field.<init>(Field.java:185) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.MysqlIO.unpackField(MysqlIO.java:723) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.MysqlIO.getResultSet(MysqlIO.java:418) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.MysqlIO.readResultsForQueryOrUpdate(MysqlIO.java:3112) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.MysqlIO.readAllResults(MysqlIO.java:2341) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2736) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2490) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1858) ~[patched_1.12.jar:git-Paper-1168]
    at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1966) ~[patched_1.12.jar:git-Paper-1168]
    at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeQuery(ProxyPreparedStatement.java:52) ~[?:?]
    at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeQuery(HikariProxyPreparedStatement.java) ~[?:?]
    at me.bimmr.bimmcore.mysql.MySQLManager.handleSQL(MySQLManager.java:223) ~[?:?]
    at me.bimmr.bimmcore.mysql.MySQLManager.querySQL(MySQLManager.java:264) ~[?:?]
    at me.bimmr.bimmcore.mysql.MySQLManager.get(MySQLManager.java:114) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:115) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.getPlayersRank(RankManager.java:126) ~[?:?]
    at com.bimmr.mcinfectedranks.Ranks.RankManager.setPlayersRank(RankManager.java:149) ~[?:?]
```

----
## Your Environment

* Spigot Version used: 1.12
* McInfected Version: 1.5.1
* BimmCore Version: 0.0.8
* Any Addons?(Include their versions) ranks, items, dedicated

**Comment by Bimmr** [comment]
I'll look into it this weekend, I have an idea of what's causing it.

**Comment by Bimmr** [comment]
Fixed in McInfected Ranks - [v1.2.4]
